tools/static/hooks/hook-pycbc.py

```diff
@@ -57,7 +57,9 @@ def find_lib_path(libname, packages):
                  'h5py._conv',
                  'h5py._stub',
                  'mpld3',
-                 'M2Crypto'
+                 'M2Crypto',
+                 'lal',
```

Comment:
What is lal here? If it's the swig libraries, we shouldn't need this.

Reply:
@ahnitz, @spxiwh, **`lscsoft-glue>=1.56` does not depend on `M2Crypto`**, and correctly declares its own dependencies on `six`, `pyOpenSSL`, and `numpy`.

Reply:
* I think lal was added here to try to fix things that are now fixed in glue. So I've reverted the change.

 * @duncanmmacleod Good point about M2Crypto, I've removed all references to this from PyCBC's build test stuff.